sampleSpecs/defaults.smithy

```diff
@@ -11,3 +13,10 @@ structure DefaultTest {
   two: String = "test"
   three: StringList = []
 }
+
+structure DefaultOrderingTest {
```

Comment:
Mmm, I think it might be a good opportunity to add unit tests in the `codegen` module, since relying on our usual smoke-test way of doing things ain't gonna work (as metadata is meant to be unique across a smithy model). 

Would you mind adding such a unit test, a little bit like the smithy-translate ones, where we dynamically parse a model and run it through the codegen pipeline, and assert that the output code is what we think it should be ?

Reply:
Yeah that's a good idea, will add 👍